contents/handbook/growth/sales/sales-operations.md

```diff
@@ -6,120 +6,66 @@ showTitle: true
+- _Low volume, more technical_ - Open Source will be fine up to 10k MAUs. Beyond that, the product will still work but Postgres limitations at scale means performance will be degraded. 
+- _High volume, less technical_ - Cloud will be the best bet - pricing does increase at scale as we take on hosting costs, but the setup process and ongoing maintenance is very straightforward. 
+- _High volume, more technical_ - either Scale or Cloud. Scale will be more appealing for customers who need to keep control of their data for privacy or corporate policy reasons. The pricing for Scale is greatly discounted at higher volumes vs. Cloud as we don't pay hosting costs. 
 
-## Managing our CRM
+### What about Scale Free?
 
-People currently come into HubSpot through one of 3 ways:
-- They email hey@posthog.com, sales@posthog.com or another email address if we have created a custom one for a specific group
-- They sign up to the PostHog app
-- They are manually added to HubSpot by a member of the team, e.g. if you met someone interested in PostHog at an event
+We have recently started rolling out Scale Free. Our plan is to make it widely available as a config option in Open Source, but we're keeping them separate for now. Scale Free will be appealing to customers who need Clickhouse due to volume rather than Postgres, but are happy with 3 logins only and community-based support. 
 
-### Email
+We have a short waiting list of customers waiting for deployment, so are currently prioritising those with 10k-1m MAUs, are on GCP and are familiar with Helm Charts/k8s. 
 
-New conversations come into 'Unassigned', whereas ongoing conversations will go straight to your inbox.
+## FAQs
 
-We do not have super defined roles here, but generally:
-- James H deals with Enterprise queries
-- Yakko takes care of Startup queries
-- Paolo focuses on existing customers
-- Charles oversees sales ops and HubSpot admin
+_Can I give a Scale customer a free trial?_
 
-However, anyone can and should jump in if they can help or they see someone hasn't been responded to, especially when folks are on holiday!
+No, because we don't need to - they can get up and running with our Scale Free or Cloud plans first if they want to try out PostHog for free. You'll find a lot of inbound customers will do this anyway before talking to us about Scale. 
 
-We have lots of handy templates you can use as well - just select _Templates_ in the email window in Hubspot. If you find yourself sending the same type of email repeatedly, you may want to create your own template - go to 'Conversations' -> 'Templates'.
+_Can I give a Scale customer a discount?_
 
-If an inbound email is about one of our Startup or Enterprise plans, you should create a Deal - more on this below. 
+Again, no need - we already have usage-based pricing which is _heavily_ discounted at higher volumes, and we only bill month-to-month, so customers don't need to feel locked in to a longer term contract. 
```

Comment:
I'd add:

With extremely high volume users, they may want to lock in the contract for a longer term so we can't ie triple our pricing overnight. We'd look to do this with people spending $10k/month or more - we'd move them onto a more traditional Master Services Agreement. We don't do it below this value because of the legal effort required.